docs/wireframe/09-mypage.html

```
@@ -0,0 +1,61 @@
+ㄹ<!DOCTYPE html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)